consumer/core/src/main/java/org/arquillian/algeron/consumer/core/ContractsPublisher.java

```diff
@@ -42,7 +42,7 @@ public void publish(@Observes AfterClass event, AlgeronConsumerConfiguration alg
             }
         }
 
-        throw new IllegalArgumentException(String.format("No pact publisher registered with name %s.", name));
+        throw new IllegalArgumentException(String.format("No contract publisher registered with name %s.", name));
```

Comment:
My thinking is - argument refers to the argument passed to the method. Hence my question :)

To quote JDK

> Thrown to indicate that a method has been passed an illegal or inappropriate argument.

At this point - no provider name registered - I guess we are talking about the state.